cpp/setup.py

```diff
@@ -1,69 +0,0 @@
-import os
-import re
-import sys
-import platform
-import subprocess
-
-from setuptools import setup, Extension
```

Comment:
I think these classes should go inside the python/twisterx directory

Reply:
Yes, this was removed. That is true.